### PR TITLE
Add missing labels.* and annotations.* fields to kubernetes package

### PR DESCRIPTION
**Pull request by tetianakravchenko**
Signed-off-by: Tetiana Kravchenko <tetiana.kravchenko@elastic.co>

## What does this PR do?
- Add missing `labels.*` and `annotations.`* fields to kubernetes package resources: pod, container and service
- update sample_event

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes https://github.com/elastic/integrations/issues/2351

### CODE REVIEW

File: packages/kubernetes/manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: kubernetes
 title: Kubernetes
-version: 1.8.0
+version: 1.8.1
```
Comment (ChrsMark) [inline packages/kubernetes/manifest.yml:4]: I would say `1.9.0` and upgrade the stack version of the package to `8.1.0`. We need to confirm though with ecosystem team that's it's fine to remove the 7.16.0 constraint.

Comment (ChrsMark) [reply packages/kubernetes/manifest.yml:4]: Update on this. We can leave the constraint as is for now, 7.16.0 + 8.0.0

Comment (MichaelKatsoulis) [reply packages/kubernetes/manifest.yml:4]: +1 for the 1.9.0 but I don't think we should change the Kibana constraints. The older versions won't report those fields. This won't create any issue.If we remove the 7.16.0 then we won't be able to deliver bugfixes like https://github.com/elastic/integrations/pull/2288

Comment (tetianakravchenko) [reply packages/kubernetes/manifest.yml]: I've changed version to `1.9.0` https://github.com/elastic/integrations/pull/2361/commits/5ec5290d71e128218b3c8b99120a20994a61ffe4

**Review by ChrsMark — APPROVED** [review]
LGTM, let's wait for CI

**Comment by MichaelKatsoulis** [comment]
I merged https://github.com/elastic/integrations/pull/2288 so you will have to resolve the version conflicts first

**Review by MichaelKatsoulis — APPROVED** [review]

**Pull request merged**